client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -0,0 +1,350 @@
+					<div className="woocommerce-admin__business-details__selective-extensions-bundle__extension">
+						<CheckboxControl
+							id="woocommerce-business-extensions__checkbox"
+							checked={ values.install_extensions }
+							onChange={ ( checked ) => {
+								setValues(
+									setAllPropsToValue( values, checked )
+								);
+							} }
+						/>
+						<p className="woocommerce-admin__business-details__selective-extensions-bundle__description">
+							{ __(
+								'Add recommended business features to my site'
+							) }
+						</p>
+						<Icon
+							className="woocommerce-admin__business-details__selective-extensions-bundle__expand"
+							icon={ showExtensions ? chevronUp : chevronDown }
+							onClick={ () => {
+								const shouldShowExtensions = ! showExtensions;
+								setShowExtensions( shouldShowExtensions );
+
+								if ( shouldShowExtensions ) {
+									// only record the accordion click when the accordion is opened.
+									recordEvent(
+										'storeprofiler_store_business_features_accordion_click'
+									);
+								}
+							} }
+						/>
+					</div>
+					{ showExtensions &&
+						installableExtensions.map( ( { plugins, title } ) => (
+							<div key={ title }>
+								<div className="woocommerce-admin__business-details__selective-extensions-bundle__category">
+									{ title }
+								</div>
+								{ plugins.map( ( { description, slug } ) => (
+									<div
+										key={ slug }
+										className="woocommerce-admin__business-details__selective-extensions-bundle__extension"
+									>
+										<CheckboxControl
+											id="woocommerce-business-extensions__checkbox"
+											checked={ values[ slug ] }
+											onChange={ ( checked ) => {
+												const newState = {
+													...values,
+													[ slug ]: checked,
+												};
+
+												const allExtensionsDisabled =
+													Object.entries(
+														newState
+													).filter(
+														( [ , val ] ) => val
+													).length === 1 &&
+													newState.install_extensions;
+
+												if ( allExtensionsDisabled ) {
+													// If all the extensions are disabled then disable the "Install Extensions" checkbox too
+													setValues( {
+														...newState,
+														install_extensions: false,
+													} );
+												} else {
+													setValues( {
+														...values,
+														[ slug ]: checked,
+														install_extensions: true,
+													} );
+												}
+											} }
+										/>
+										<p className="woocommerce-admin__business-details__selective-extensions-bundle__description">
+											{ description }
+										</p>
+										<FreeBadge />
+									</div>
+								) ) }
+							</div>
+						) ) }
+				</div>
+				<div className="woocommerce-profile-wizard__business-details__free-features__action">
+					<Button
+						onClick={ () => {
+							onSubmit( values );
```

Comment:
the values from the form are passed back up to the parent component by `onSubmit`